docs/source/intro/run_docker.rst

```diff
@@ -19,29 +19,19 @@ This image contains a fully pre-configured AiiDA environment which makes it part
 
       To install Docker, please refer to the `official documentation <https://docs.docker.com/get-docker/>`__.
 
-      .. note::
-
-         If you are using Linux, you need to have root privileges to do `post-installation steps for the Docker Engine <https://docs.docker.com/engine/install/linux-postinstall/>`__.
```

Comment:
This is not required anymore since for the linux the official docker doc also recommend to use docker desktop.

Reply:
I think it is fine to have the latest docs to point to a release that does not exist yet. We will be releasing soon.